webui/react/src/pages/TrialDetails.tsx

```diff
@@ -71,11 +72,21 @@ const trialContinueConfig = (
   };
 };
 
+const STORAGE_PATH = 'trial-detail';
+const STORAGE_CHECKPOINT_VALIDATION_KEY = 'checkpoint-validation';
+const STORAGE_METRICS_KEY = 'metrics';
+
 const TrialDetailsComp: React.FC = () => {
   const { trialId: trialIdParam } = useParams<Params>();
   const trialId = parseInt(trialIdParam);
+  const [ trialResponse, triggerTrialRequest ] =
+    useRestApi<TrialDetailsParams, TrialDetails>(getTrialDetails, { id: trialId });
   const [ experiment, setExperiment ] = useState<ExperimentDetails>();
-  const [ showHasCheckpoint, setShowHasCheckpoint ] = useState(true);
+  const storage = useStorage(STORAGE_PATH);
+  const trialStorage = useStorage(`${STORAGE_PATH}/${trialResponse?.data?.experimentId}`);
```

Comment:
how about `storage_path/experiments/1` or `/experiments/1/trial-detail` ? 

so that the paths can easily show what their scope is 

![Screen Capture_select-area_20200826170122](https://user-images.githubusercontent.com/12127420/91368441-f09ede00-e7bd-11ea-86b0-cb72e855fb3e.png)

Reply:
`${storage_path}/experiments/1` sounds good to me!